### ISSUE TITLE
get_data rounds off REAL values

### ISSUE THREAD

**Issue by joaquinvanschoren** [openml/openml-python#197]
If you run this:
```
liver = oml.datasets.get_dataset(8) # Download Liver-disorders data
X, y = liver.get_data(target=liver.default_target_attribute);
y
```
You'll see that all targets y are integers. 
The actual target is a REAL number, and gets rounded!

**Comment by mfeurer** [comment]
Try `X, y = liver.get_data(target=liver.default_target_attribute, target_dtype=float)`. In the long run I suggest that you use tasks in order to retrieve the datasets you're looking at. Tasks usually handle this themselves.

**Comment by joaquinvanschoren** [comment]
Thanks! Would be good to add that to the documentation, though?

**Comment by mfeurer** [comment]
How about making this function private, and only exposing this correctly via the task object? People should rarely have to access raw data without a task.

Thoughts @joaquinvanschoren @janvanrijn @amueller

**Comment by janvanrijn** [comment]
Agreed.

2017-10-09 17:44 GMT+02:00 Matthias Feurer <notifications@github.com>:

**Comment by amueller** [comment]
closed by #352
